### ISSUE TITLE
Oracle lastInsertId returning array - breaks saving related records

### ISSUE THREAD

**Issue by Skydev0h** [phalcon/cphalcon#11862]
Oracle lastInsertId returns array of value instead of the actual value.
This breaks saving of related records.
![image](https://cloud.githubusercontent.com/assets/5331782/15905728/98815c38-2dbe-11e6-8874-a3be4765561f.png)

To fix you need to change line `228` of file `phalcon/db/adapter/pdo/oracle.zep` to this:
```PHP
return this->fetchAll("SELECT " . sequenceName . ".CURRVAL FROM dual", \Phalcon\Db::FETCH_NUM)[0][0];
```
instead of
```PHP
return this->fetchAll("SELECT " . sequenceName . ".CURRVAL FROM dual", \Phalcon\Db::FETCH_NUM)[0];
```

**Comment by Jurigag** [comment]
If you know how to create fix then what about doing PR ?

**Comment by tembem** [comment]
@Skydev0h i help you creating this pull request https://github.com/phalcon/cphalcon/pull/11874

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch.
